specs/interop/upgrade.md

```diff
@@ -30,6 +33,43 @@ The execution payload MUST set `noTxPool` to `true` for this block.
 
 The exact definitions for these upgrade transactions are still to be defined.
 
+## L1 Attributes
+
+On the Interop activation block, and if Interop is not activated at Genesis,
+the L1 Attributes Transaction includes a call to `setL1BlockValuesEcotone`.
+
+Every subsequent L1 Attributes transaction should include a call to the new `setL1BlockValuesInterop` function.
```

Comment:
nit: we renamed this to `setL1BlockValuesIsthmus` right?

Reply:
oops yes! nice catch.